azure/storage/constants.rb

```diff
@@ -27,6 +27,13 @@ module Service
       module Defaults
         LOGGER_LEVEL = Logger::DEBUG
       end
+
+      module Constant
+        TICKSPERSECOND = 10000000
```

Comment:
Instead of `TICKSPERSECOND`, how about `TICKS_PER_SECOND`, cause ruby and PHP encourages capital with underscores for constants. See [here](https://en.wikipedia.org/wiki/Naming_convention_(programming))